interface/patient_tracker/patient_tracker.php

```diff
@@ -138,400 +146,499 @@ function refreshbegin(first){
     setTimeout("refreshbegin('0')",parsetime);
   <?php } else { ?>
     return;
- <?php } ?>
-}
+    <?php } ?>
+  }
 
 // used to display the patient demographic and encounter screens
 function topatient(newpid, enc) {
- if (document.pattrk.form_new_window.checked) {
-   openNewTopWindow(newpid,enc);
- }
- else {
-   top.restoreSession();
-   if (enc > 0) {
-     top.RTop.location= "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + newpid + "&set_encounterid=" + enc;
-   }
-   else {
-     top.RTop.location = "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + newpid;
-   }
- }
+  if (document.pattrk.form_new_window.checked) {
+    openNewTopWindow(newpid,enc);
+  }
+  else {
+    top.restoreSession();
+    if (enc > 0) {
+      top.RTop.location= "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + newpid + "&set_encounterid=" + enc;
+    }
+    else {
+      top.RTop.location = "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + newpid;
+    }
+  }
 }
 
+    function clearCALLback(eventdate,eid,pccattype) {
+      if (confirm('<?php echo xla("This patient has requested a call from the office"); ?>. <?php echo xla('Are you ready to remove this flag'); ?>?\n<?php echo xla("If you have resolved the issue and you are ready to delete this flag from the Flow Board, select OK"); ?>.')) {
```

Comment:
use xls in above line rather than xla (xls uses addslashes rather than htmlspecialchars since this is a javascript literal)